java/src/org/openqa/selenium/grid/commands/Standalone.java

```diff
@@ -187,6 +187,12 @@ protected Handlers createHandlers(Config config) {
         new GraphqlHandler(
             tracer, distributor, queue, serverOptions.getExternalUri(), getFormattedVersion());
 
+    Boolean disableUi = new RouterOptions(config).disableUi();
+    if (disableUi) {
+      Node node = createNode(config, bus, distributor, combinedHandler);
+      return new Handlers(graphqlRoute("", () -> graphqlHandler), new ProxyNodeWebsockets(clientFactory, node));
```

Comment:
Same concern here about returning the route here.

Reply:
Thanks @pujagani for the feedback, I made the changes based on this.